conference/react-js/src/containers/SignInForm/index.jsx

```diff
@@ -8,7 +8,7 @@ import { bindActionCreators } from 'redux';
 import loadData from '../../actions/load';
 import { forgotPassword, signUp } from '../../constants/route-url';
 import login from '../../actions/login';
-import getUserInfo from '../../actions/getUserInfo';
+import getUserInfo from '../../actions/get-user-info';
```

Comment:
Aren't you use camel case?

Reply:
I think camel case is not good at file names, maybe we need to talk about it...